src/Files.App/Constants.cs

```diff
@@ -234,6 +234,8 @@ public static class UserEnvironmentPaths
 
 			public static readonly string TempPath = Environment.GetEnvironmentVariable("TEMP") ?? "";
 
+			public static readonly string SystemDrivePath = Environment.GetEnvironmentVariable("SystemDrive") ?? "C:";
```

Comment:
You may want to consider the case where the environment variable is an empty string.

Reply:
Implemented in [8512bc4](https://github.com/files-community/Files/pull/15842/commits/8512bc47c239528a7bf1d6f59148cafb75fb489e)